changelogs/2024-03-14_new-keys-to-allow-and-remove-custom-markup.mdx

```diff
@@ -0,0 +1,50 @@
+
+This release addressed several issues, including those affecting publishing to posts in Jekyll, image duplication in the Visual Editor, embeds, the conflicting updates warning, and new files.
+
+### Features & Improvements:
+
+* Added more options for handling HTML or Markdown structure that cannot be recreated in the rich text editor.
+  * The `allow_custom_markup` key allows you to edit custom markup in rich text editors and explicitly accept the risk of unintentionally deleting elements that cannot be recreated.
+  * The `remove_custom_markup` key will strip custom markup from content edited in a rich text editor, allowing you to clean files of unnecessary HTML content.
+  * The `remove_custom_markup` key is disabled if `allow_custom_markup` is enabled.
+  * You can define these keys anywhere you would configure a rich text toolbar. By default, `allow_custom_markup` is `false` for inputs and `true` for editable regions, and `remove_custom_markup` is `false` for inputs and editable regions.
+  * You can configure both keys using the configuration GUI.
+  * When using `allow_custom_markup` outside of editable regions, content that uses JavaScript will not be editable for security reasons. These inputs will appear as uneditable snippets.
+  * For more information, please read our documentation on best practices for HTML in [rich text](/documentation/articles/best-practices-for-rich-text/) and [editable regions](/documentation/articles/define-editable-regions-in-your-html/).
+* You can now add documentation links to checkboxes and switches.
+* You can now copy your build log using the *Copy output* button in the build modal, on the *Status* page, and in the *Builds* history tab.
+* You can now specify a custom URL for your dashboard preview screenshot using `editor.default_path`.
+* You can now use nested keys for any key inside `preview`.
+* The&nbsp;*View output*&nbsp;button is now available while your build is optimizing assets.
+* The embeds tool in the&nbsp;*WYSIWYG toolbar*&nbsp;now opens a code editor instead of a text area.
+* Invalid editable regions will now preserve their original content and have better error messaging.
+
+### Fixes:
+
+* Fixed an issue where cloning or renaming a file from within an editing interface would not immediately open the cloned or renamed file.
+* Fixed an issue where, on a Jekyll site, publishing a file to posts would not prompt you to add a date to the filename.
+* Fixed an issue where, in the Visual Editor, updating the details for an image would cause the image panel to close.
+* Fixed an issue where the&nbsp;*Context menu*&nbsp;for array and object inputs would show "Add undefined" instead of "Add item" for structures with no label configured.
+* Fixed a spacing issue affecting collapsed card lists.
```

Comment:
I think "cards" are a concept mentioned in the docs, but maybe not "collapsed"? It might be more comprehensible to just say "arrays in the Data Editor".